src/commonMain/kotlin/app/cash/turbine/Turbine.kt

```diff
@@ -211,6 +211,7 @@ internal class ChannelTurbine<T>(
     var cause: Throwable? = null
     while (true) {
       val event = channel.takeEventUnsafe() ?: break
+      if (event is Event.Error && event.throwable is CancellationException) break
```

Comment:
I really like the idea of just eating these at the source. I had dealt with `CancellationException` in an ad-hoc manner when implementing `turbineScope`, and this is much cleaner